packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -1293,6 +1291,7 @@ space
 
       it('is case sensitive when matchCase is undefined', () => {
         cy.get('#test-button').contains('Test')
+        cy.contains('test').should('not.exist')
```

Comment:
Adding this assertion because I had problems show up in Launchpad component tests due to incorrect handling of case in .contains(). I thought I had the casing correct because we had tests covering it, but turns out they only asserted the positive case and not the negative.